deploy/rbd/kubernetes/v1.13/csi-rbdplugin.yaml

```diff
@@ -7,8 +7,6 @@ spec:
   selector:
     matchLabels:
       app: csi-rbdplugin
-  updateStrategy:
```

Comment:
Should we leave a note here (and in the ` deploy/rbd/kubernetes/v1.14+/csi-rbdplugin.yaml` file), like the helm charts stating "# if you are using [rbd-nbd|cephfs-fuse] client set this value to OnDelete"?

Reply:
This is taken care in upgrade documentation  https://github.com/ceph/ceph-csi/pull/770